src/SprykerEco/Zed/PunchoutCatalogs/Communication/Controller/IndexController.php

```diff
@@ -38,4 +50,116 @@ public function tableAction(): JsonResponse
 
         return $this->jsonResponse($table->fetchData());
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return array|\Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function createAction(Request $request)
+    {
+        $punchoutCatalogConnectionsForm = $this->getFactory()
+            ->getPunchoutCatalogConnectionForm();
+
+        $punchoutCatalogConnectionsForm->handleRequest($request);
+
+        if ($punchoutCatalogConnectionsForm->isSubmitted() && $punchoutCatalogConnectionsForm->isValid()) {
+            $punchoutCatalogResponseTransfer = $this->getFacade()
+                ->createConnection($punchoutCatalogConnectionsForm->getData());
+
+            if ($punchoutCatalogResponseTransfer->getIsSuccessful()) {
+                $this->addSuccessMessage('Connection added');
```

Comment:
By message itself (here and further), don't you think that just `Connection` is not specific enough? I would use the full name of the entity, like `PunchOut connection` (... added, in the current case).

Reply:
I would leave it as because here is the least of initial translations that were present in the branch initially (I've moved from glossary file) https://github.com/spryker/suite-nonsplit/pull/2248/files